src/findpython/python.py

```diff
@@ -11,7 +12,12 @@
 from findpython.utils import get_binary_hash
 
 logger = logging.getLogger("findpython")
-GET_VERSION_TIMEOUT = 5
+_DEFAULT_GET_VERSION_TIMEOUT = 5
+
+
+def get_version_timeout() -> float:
```

Comment:
Why do we getting the value on-the-fly instead of freezing it when start up?

Reply:
That’s what I generally do with these sorts of environment, variables to avoid issues with sequencing of setting and reading. But happy to make it a global if you’d prefer.

Reply:
Yes please do, lack of practical case and inefficient function call

Reply:
done